src/backend/db/videos.js

```diff
@@ -30,8 +30,8 @@ export const videos = [
     speaker: "Master Shi Heng Yi",
     description:
       "Meet Shaolin Master Shi Heng Yi in his serene talk about self-discovery. Learn why rainfall is an essential part of each flowering. And every small step – part of the journey to the highest peek. The hindrances along the way to self-discovery and personal growth are easy to overcome. Learn how from his talk.",
-    categoryName: "Mashup",
-    img: "https://i.ytimg.com/vi/4-079YIasck/0.jpg",
+    categoryName: "mastery",
+    img: "https://img.youtube.com/vi/4-079YIasck/maxresdefault.jpg",
```

Comment:
instead of adding image link for each image in database you can directly add this link in video card and just loop over id